### DISCUSSION TITLE
Removed create-remix options for deployment?

### DISCUSSION THREAD

**Discussion by janpio** [remix-run/remix#8506] [Q&A]
For example https://about.gitlab.com/blog/2022/11/21/deploy-remix-with-gitlab-and-cloudflare/ mentions that `create-remix` would be asking for deployment options:

```
? Where do you want to deploy? [...]
> choose "Cloudflare Workers"
```

When replicating the commands myself, I do not get that question.
Has it been removed?
What command or process has replaced it?

**Comment by sergiodxa** [comment]
The list of templates where removed from the CLI, you can use the `--template` option to point it any template, for Cloudflare Workers you can use:

```sh
npx create-remix --template remix-run/remix/templates/cloudflare-workers
```

If you're going to use Cloudflare, I recommend you more to use Cloudflare Pages

```sh
npx create-remix --template remix-run/remix/templates/cloudflare-pages
```

Finally, you can find the docs of the `create-remix` CLI here https://remix.run/docs/en/main/other-api/create-remix, it includes a section on how to use the `--template` command, the list of official templates is on GitHub here https://github.com/remix-run/remix/tree/main/templates.

**Reply by janpio** [reply]
Thanks, that makes a lot of sense.

I tried to piece that together from the docs as well, but only found bits and pieces, and never the full story.

> If you're going to use Cloudflare, I recommend you more to use Cloudflare Pages

Can you elaborate on that? I also already searched for that, but again didn't find much explanation, just bits and pieces.
